### ISSUE TITLE
Poi 10.2.0 not processing stylus files imported in index.js (for Vuetify a la carte)

### ISSUE THREAD

**Issue by abelovic** [egoist/poi#444]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

I'm getting the following when upgrading to POI 10.2.0. I'm also using vuetify a la carte like this: https://vuetifyjs.com/en/guides/a-la-carte. It appears as if it is no longer processing the stylus files when imported via index.js. To compare I tried import a stylus file in app.vue using the following:

`<style lang="stylus" src="../assets/stylus/transitions.styl" />`

The above worked. But when using index.js, I get this (many more but omitted for brevity)

`Module parse failed: Unexpected character '@' (1:0)
You may need an appropriate loader to handle this file type.
| @import '../bootstrap'
|
| .avatar

Module parse failed: Unexpected character '@' (1:0)
You may need an appropriate loader to handle this file type.
| @import '../bootstrap'
|
| .badge`

**If the current behavior is a bug, please provide the steps to reproduce.**

Seems to happen only for stylus when importing files in the index.js (which I need for vuetify a la  carte)

**Please mention other relevant information such as the browser version, Node.js version, Poi version and Operating System.**

Windows 10
Node 8.11.1
Vuetify: 1.0.18
Vue: 2.5.16

  "devDependencies": {
    "@babel/core": "7.0.0-beta.49",
    "@babel/polyfill": "^7.0.0-beta.49",
    "@babel/preset-stage-2": "^7.0.0-beta.49",
    "@poi/plugin-bundle-report": "^10.0.1",
    "@poi/plugin-offline": "^10.0.1",
    "babel-plugin-transform-imports": "^1.5.0",
    "babel-preset-poi": "10.0.1",
    "copy-webpack-plugin": "^4.5.1",
    "exports-loader": "^0.7.0",
    "node-sass": "^4.9.0",
    "offline-plugin": "^5.0.3",
    "poi": "^10.2.0",
    "sass-loader": "^7.0.1",
    "style-loader": "^0.21.0",
    "stylus": "^0.54.5",
    "stylus-loader": "^3.0.2",
    "uglifyjs-webpack-plugin": "^1.2.5",
    "vue-template-compiler": "^2.5.16",
    "webpack": "^4.9.1"
  }

**Comment by lgraziani2712** [comment]
I have the same error but with SASS:

```sh
× Failed to compile with 1 errors

Module build failed: Unknown word (5:1)

  3 | // load the styles
  4 | var content = require("!!../node_modules/css-loader/index.js??ref--1-oneOf-0-1!../node_modules/sass-loader/lib/loader.js??ref--1-oneOf-0-2!./app.scss");
> 5 | if(typeof content === 'string') content = [[module.id, content, '']];
    | ^
  6 | if(content.locals) module.exports = content.locals;
  7 | // add the styles to the DOM
  8 | var add = require("!../node_modules/vue-style-loader/lib/addStylesClient.js").default
```

**Comment by lgraziani2712** [comment]
Indeed, I could fix it moving everything to the vue component

**Comment by egoist** [comment]
Fixed in 10.2.1
